app/assets/javascripts/controllers/ansible_credentials/ansible_credentials_form_controller.js

```diff
@@ -0,0 +1,130 @@
+ManageIQ.angular.app.controller('ansibleCredentialsFormController', ['$http', '$window', 'credentialId', 'miqService', 'API', function($http, $window, credentialId,  miqService, API) {
+  var vm = this;
+
+  var init = function() {
+    vm.credentialModel = {
+      id: null,
+      name: '',
+      type: '',
+    };
+
+    vm.credential_options = {};
+    vm.select_options = [];
+
+    vm.newRecord = credentialId == 'new';
```

Comment:
Recommended syntax is --

`vm.newRecord = credentialId === 'new';`
(note that extra`=`)